lib/phoenix/socket/transport.ex

```diff
@@ -388,6 +388,57 @@ defmodule Phoenix.Socket.Transport do
     end
   end
 
+  @doc """
+  Extracts connection information from the conn and return a map.
+
+  Keys are retrieved from the optional transport option `:connect_info`.
+  This functionality is transport specific. Please refer to your transports's
```

Comment:
I think it should be “transport’s”

Reply:
I believe it is correct. Or it would also be `transports'`.

Reply:
Ah! I thought that was singular. I think it should be then `transports’`. Luckily, we have some native English speakers reviewing this PR 🙂